Probe/receive

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Receive messages from Android app
+
+android_path=./sdcard/Android/data/com.google.firebase.messaging.testing.fcmexternalprobertarget/files/
+
+file_name=$1
+shift
+
+while [[ $1 != "" ]]; do
```

Comment:
Add a comment, because I have no clue what this is doing

Reply:
addressed in 765a7ee